### PR TITLE
Feature/959 3

### PR DESCRIPTION
**Pull request by RomanGalochkin**
I have done it with direct deleting from DB but I do not like it. The best way is to mark "notebook" as "deletedAt".

### CODE REVIEW

File: packages/client/src/contexts/Lang.re [hunk 1]
```diff
@@ -15,4 +15,5 @@ let measurableIsSaved = "Question is saved.";
 let copiedMessage = "Copied";
 let copiedDescription = "The question ID was copied.";
 let backButtonTip = "Back";
-let pageIsNotFound = "Sorry, but the page is not found.";
\ No newline at end of file
+let pageIsNotFound = "Sorry, but the page is not found.";
+let notebookRemove = "Are you sure delete this notebook?";
```
Comment (OAGr) [inline packages/client/src/contexts/Lang.re]: Change to "Are you sure you want to delete this notebook?"

File: packages/client/src/contexts/Lang.re [hunk 2]
```diff
@@ -15,4 +15,5 @@ let measurableIsSaved = "Question is saved.";
 let copiedMessage = "Copied";
 let copiedDescription = "The question ID was copied.";
 let backButtonTip = "Back";
-let pageIsNotFound = "Sorry, but the page is not found.";
\ No newline at end of file
+let pageIsNotFound = "Sorry, but the page is not found.";
```
Comment (OAGr) [inline packages/client/src/contexts/Lang.re]: Change to "Page not found"

File: packages/client/src/contexts/Lang.re (hunk 1 above)
Comment (RomanGalochkin) [reply packages/client/src/contexts/Lang.re]: Thanks.

**Comment by OAGr** [comment]
Great, will merge after it passes Travis.

**Comment by OAGr** [comment]
> I have done it with direct deleting from DB but I do not like it. The best way is to mark "notebook" as "deletedAt".

I semi-agree. I think there are some nicities of actually deleting the data; for the sake of allowing the user the ability to make sure that we don't store it anywhere. 

In the future I'd be find changing this to "archive".

That said, for Notebooks, I'm not too worried, because it's easy enough just to copy the main markdown content.

**Pull request merged**